easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb

```diff
@@ -0,0 +1,22 @@
+name = 'pgi'
```

Comment:
I am happy with using PGI internally everywhere, but leaving the option open to use a "pgi" lowercase modulename, as the PGI compiler supplies itself.

I am in a bit of a crunch now trying to get our easybuild structure usable by users for testing (see my list email to come about lmod lua files) -- I will have more time next week if that is ok.